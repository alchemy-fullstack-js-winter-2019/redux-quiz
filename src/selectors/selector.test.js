import { getTitle, getSubtitle } from './selector';

describe('selector test', () => {
  it('selector', () => {
    const state = {
      title: 'i hate this test'
    };
    const updatedTitle = getTitle(state);
    expect(updatedTitle).toMatchSnapshot();
  });
});