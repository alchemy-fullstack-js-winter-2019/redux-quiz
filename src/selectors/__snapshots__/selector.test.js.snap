// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selector test selector 1`] = `"i hate this test"`;

exports[`selector test selector 2`] = `[Function]`;
